### PR TITLE
Add test runner option to repeat tests.

### PR DESCRIPTION
**Pull request by brendandahl**
Helpful with running flaky tests many times.

### CODE REVIEW

**Review by sbc100 — APPROVED** [review]
Fair enough.   I think you can also just do this with bash with `for run in {1..10}; do command; done` but maybe this is easier.

File: test/runner.py [hunk 1]
```diff
@@ -389,6 +390,8 @@ def parse_args(args):
                            'Useful when combined with --failfast')
   parser.add_argument('--force64', action='store_const', const=True, default=None)
   parser.add_argument('--crossplatform-only', action='store_true')
+  parser.add_argument('--repeat', type=int, default=1,
+                      help='Repeat each test N times (default: 1).')
```
Comment (sbc100) [inline test/runner.py:394]: Won't `--help` add the default automatically (i.e. you should n't need to write it out in the help I think).

Comment (brendandahl) [reply test/runner.py:394]: I don't see a default with `--help`

**Comment by brendandahl** [comment]
> Fair enough. I think you can also just do this with bash with `for run in {1..10}; do command; done` but maybe this is easier.

It's a little nicer for browser tests since it doesn't close the browser and restart between each run.

**Comment by sbc100** [comment]
> > Fair enough. I think you can also just do this with bash with `for run in {1..10}; do command; done` but maybe this is easier.
> 
> It's a little nicer for browser tests since it doesn't close the browser and restart between each run.

True!  I thought we actually have this feature already somewhere but I don't see it now.

lgtm!

**Pull request merged**